Adding additional instructions to the Docker Engine installation/inde…

mikebrow opened the pull request:
Docs update per discussions with @moxiegirl regarding #15823

Signed-off-by: Mike Brown brownwm@us.ibm.com

mikebrow committed:
Adding additional instructions to the Docker Engine installation/index page per discussions with @moxiegirl.

Removed the "Other Linux" heading; Changed the "Next Steps" heading to "Where to go After Installing"

Signed-off-by: Mike Brown <brownwm@us.ibm.com>

moxiegirl commented on docs/installation/index.md:
```diff
@@ -1,33 +1,49 @@
 * [openSUSE and SUSE Linux Enterprise](SUSE)
 * [Ubuntu](ubuntulinux)
+
+## Other Linux
```
@mikebrow Thanks for doing this.  Looks great.  I'd remove this heading `## Other Linux`

mikebrow replied:
The binaries are sort of different than all the others... off the reservation sort of thing.  How best to separate it from the other linux installs?

moxiegirl replied:
The new paragraph is enough.  You have a list of what we have...and this is   stated as an alternative to the list. That is why the heading is not needed.

thaJeztah replied:
Agreed, I think the heading is redundant

mikebrow replied:
ok :-) will do

moxiegirl commented on docs/installation/index.md:
```diff
@@ -1,33 +1,49 @@
+Instructions for installing prior releases of Docker can be found in the following docker archives:
+[Docker v1.7](http://docs.docker.com/v1.7/), [Docker v1.6](http://docs.docker.com/v1.6/), [Docker v1.5](http://docs.docker.com/v1.5/), and [Docker v1.4](http://docs.docker.com/v1.4/).
+
+## Where to go next
```
@mikebrow  We don't really need this section. If they haven't installed, it doesn't make sense to tell them to go somewhere next.

mikebrow replied:
Doesn't hurt either :-)  The other products machine, etc. have a where to go next section. It's just an emphasis to get them to get some training, read more docs after install etc.

moxiegirl replied:
It logically doesn't make sense. They haven't installed the product. The where to go next *is* the installation.  Some of our other pages have that not all of them. They are inconsistently used and so we will be coming up with some guidelines.

thaJeztah replied:
Agreed that the "next" step would be the actual installation, but won't make a fuss if the links are kept :smile:

mikebrow replied:
Right ... and the best place for those steps would be at the end of each of the install pages.. I was using the call stack approach where most folks come back to the main page after doing the sub page for downloading..  How about something like... after installing on your selected platform be sure to check out these pages for next steps:

moxiegirl commented:
ping @thaJeztah --- nice new org for this page

SvenDowideit commented:
LGTM!

thaJeztah commented:
LGTM after the "other linux" heading is removed, no *strong* opinion for the removal of the "where to go next" links.

@moxiegirl (slightly side-tracking here) I recently commented on another issue with some thoughts on improving the install docs; https://github.com/docker/docker/issues/15765#issuecomment-134408029.

mikebrow commented:
@moxiegirl comments have been addressed for the changes to the index.md file.  I've removed the "Other Linux" heading and Changed the "Next Steps" heading to "Where to go After Installing"

moxiegirl commented:
@mikebrow Thanks for your patience. LGTM
